app/uk/gov/hmrc/tariffclassificationfrontend/utils/JsonFormatters.scala

```diff
@@ -28,14 +28,30 @@ import uk.gov.hmrc.tariffclassificationfrontend.models.{CaseStatus, LiabilitySta
 
 object JsonFormatters {
 
+  case class Something(value: String)
+
+  implicit val formatSomething = Json.format[Something]
+
+  Json.toJson(Something(""))
+  Json.toJson(Map[String, Option[String]]("" -> Some("")))
```

Comment:
Looks like you copied the formatter across before I deleted this test code

Reply:
will chop that out